src/session/command-history.js

```diff
@@ -1,5 +1,21 @@
+"use strict"
 const LimitedList = require("../util/limited-list")
 const Storage = require("../storage")
+const { Math } = require("globalthis/implementation")
```

Comment:
This is not necessary, `Math` is an ambient global

Reply:
Fixed (this also broke Travis, which is fun). I think this is something from my VSCode configuration trying to fix missing imports